src/components/Homepage/events.js

```diff
@@ -48,23 +48,20 @@ const Events = ({ events }) => (
       </EventsColumn>
       <Padding top={42}>
         <ul>
-          {events
-            .filter(n => !n.node.homepageFeatured)
-            .slice(0, 3)
-            .map(({ node }) => (
-              <Li fullWidth symmetrical key={`${node.id}`}>
-                <Subtitle>
-                  <a
-                    href={node.linkToEvent}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {node.eventTitle}
-                  </a>
-                </Subtitle>
-                {format(new Date(node.date), 'MMMM DD[,] dddd')}
-              </Li>
-            ))}
+          {filterMeetups(events).map(event => (
+            <Li fullWidth symmetrical key={`${event.id}`}>
+              <Subtitle noPaddingBottom>
+                <a
```

Comment:
you can use the `ExternalAnchor` component here

Reply:
Done :)